bundles/org.openhab.binding.senechome/src/main/java/org/openhab/binding/senechome/internal/SenecHomeHandler.java

```diff
@@ -228,7 +227,7 @@ private void refresh() {
             Channel channelBatteryStateValue = getThing()
                     .getChannel(SenecHomeBindingConstants.CHANNEL_SENEC_BATTERY_STATE_VALUE);
             updateState(channelBatteryStateValue.getUID(),
-                    new DecimalType(getSenecValue(response.energy.batteryState).intValue()));
+                    new QuantityType<>(getSenecValue(response.energy.batteryState).intValue(), SmartHomeUnits.ONE));
```

Comment:
You switched the item-type to Number so you need to change this back and use DecimalType again.

Reply:
OK, done that. JARs in the opener are updated again 👍